### DISCUSSION TITLE
Next Stable Version

### DISCUSSION THREAD

**Discussion by Shinjiru-stack** [glowroot/glowroot#931] [General]
Any update on next stable version of glowroot/when can we expect it?

**Comment by varikin** [comment]
Any update on this? Would love to have a stable version instead of a beta that supports Java 17.

**Comment by nowheresly** [comment]
I think we're not far from a decent RC1. This could hopefully happens in the next few weeks.

**Comment by nowheresly** [comment]
The last version beta4 is near to be a release candidate. Feel free to test it and provide your feedback.

**Comment by Never-lab** [comment]
For anyone still subscribed: stable **0.14.x** has shipped — latest is **0.14.6** (Mar 2026). Java 17+ is supported in current releases; no need to stay on beta for that anymore.

Download: https://github.com/glowroot/glowroot/releases
